.github/workflows/release.yml

```
@@ -12,126 +12,174 @@
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: ${{ needs.version-check.outputs.version }}
+          name: ${{ needs.version-check.outputs.version }}
+          generate_release_notes: true
+          draft: false  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)